tools/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0
 Since this project is only a vessel for packaging a few binary tools, its adherence to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) is loose at best.
 
+## [0.2.1] - 2022-11-1
+
+Update bottlerocket sdk to v0.28.0
+
+### Contents
+
+- boringtun v0.4.0 was removed
+- eksctl 0.112.0
+- kubeadm v1.23.13
+- sonobuoy v0.56.4
+- eksctl-anywhere 0.11.4-21
```

Comment:
This needs a rebase after https://github.com/bottlerocket-os/bottlerocket-test-system/pull/637 merges.